CHANGELOG.next.asciidoc

```diff
@@ -99,9 +99,9 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Skipping unparsable log entries from docker json reader {pull}12268[12268]
 - Parse timezone in PostgreSQL logs as part of the timestamp {pull}12338[12338]
 - Require client_auth by default when ssl is enabled for tcp input {pull}12333[12333]
```

Comment:
This line should also be removed with the fix.

Reply:
Correct, since this PR change that logic it make sense to make sure that the changelog is clear.